tests/com.espressif.idf.core.test/src/com/espressif/idf/core/test/IDFVersionReaderTest.java

```diff
@@ -4,16 +4,19 @@
  *******************************************************************************/
 package com.espressif.idf.core.test;
```

Comment:
probably we can move this class to unittest package and leave test package for other kind of tests